nixos/modules/services/cluster/k3s/default.nix

```diff
@@ -340,6 +344,29 @@ in
       '';
     };
 
+    containerdConfigTemplate = mkOption {
+      type = with types; str;
+      default = ''
+        # Base K3s config
+        {{ template "base" . }}
+      '';
```

Comment:
Maybe it's better to not have this file by default?

Reply:
Done